Gemfile

```diff
@@ -59,7 +59,9 @@ gem 'sparql',         '~> 1.99'
 gem 'json-ld',        '~> 1.99'
 
 group :development, :test do
-  gem 'rdf-do',       '~> 1.99' # Using DataObjects for SPARQL store
-  gem 'data_objects', '~> 0.10'
-  gem 'do_sqlite3' ,  '~> 0.10'
+  #gem 'rdf-do',       '~> 1.99' # Using DataObjects for SPARQL store
+  #gem 'data_objects', '~> 0.10'
+  #gem 'do_sqlite3' ,  '~> 0.10'
+  gem 'rdf-mongo'
+  gem 'bson_ext'
 end
```

Comment:
There might be problems running specs on travis CI with a mongo dependency.  In general, mongo is a reasonable choice and one that I am open to supporting, although it's not widely adopted in DLSS and the dev-ops team have not committed to supporting it in production (someone please correct me if that's wrong).